contracts/vaults/BaseVaults/RibbonThetaVault.sol

```diff
@@ -278,6 +278,16 @@ contract RibbonThetaVault is RibbonVault, RibbonThetaVaultStorage {
         optionsPurchaseQueue = newOptionsPurchaseQueue;
     }
 
+    /**
+     * @notice Sets oToken Premium
+     * @param minPrice is the new oToken Premium
+     */
+    function setMinPrice(uint256 minPrice) external onlyOwner {
```

Comment:
We'll have to change this to be `onlyKeeper` because this function is called routinely. The owner is a multisig